### PR TITLE
Proxy protocol support (DON'T MERGE)

### LINKED ISSUE #22: Enable proxy protocol on Nginx
For being able to see client ip address when request is going via ssl router, we need to support Proxy Protocol on Nginx. Nginx added Proxy Protocol support since `1.5.12`, so it needs to be at least `1.5.12`.

Instruction how to do it:
https://www.nginx.com/resources/admin-guide/proxy-protocol/

Important that ELB should have Proxy Protocol enabled for this to work:
http://docs.aws.amazon.com/elasticloadbalancing/latest/classic/enable-proxy-protocol.html

### PR DESCRIPTION
**Pull request by yyunikov**
## Fixes #22

This PR enables proxy protocol support on Nginx.

### CODE REVIEW

File: templates/nginx.conf.j2 [hunk 1]
```diff
@@ -1,24 +1,30 @@
 # {{ ansible_managed }}
 # This template represents a termination point for a SSL certificate
 
+log_format elb_log '$proxy_protocol_addr - $remote_user [$time_local] '
```
Comment (yyunikov) [inline templates/nginx.conf.j2]: This corrects access log for us. Previously it was:
```
0.10.10.135 - - [03/Oct/2017:14:53:14 +0000] "GET /get HTTP/1.1" 200 524 "-" "Pingdom.com_bot_version_1.4_(http://www.pingdom.com/)"
10.10.12.28 - - [03/Oct/2017:14:54:13 +0000] "GET /get HTTP/1.1" 200 523 "-" "Pingdom.com_bot_version_1.4_(http://www.pingdom.com/)"
10.10.10.135 - - [03/Oct/2017:14:54:36 +0000] "GET /get HTTP/1.1" 200 454 "-" "curl/7.54.0"
10.10.10.135 - - [03/Oct/2017:14:55:13 +0000] "GET /get HTTP/1.1" 200 524 "-" "Pingdom.com_bot_version_1.4_(http://www.pingdom.com/)"
``` 

With proxy protocol:
```
188.191.36.87 - - [03/Oct/2017:18:56:45 +0000] "GET /get HTTP/1.1" 200 456 "-" "curl/7.54.0"
188.191.36.87 - - [03/Oct/2017:18:57:06 +0000] "GET /get HTTP/1.1" 200 455 "-" "curl/7.54.0"
208.64.28.194 - - [03/Oct/2017:18:57:13 +0000] "GET /get HTTP/1.1" 200 525 "-" "Pingdom.com_bot_version_1.4_(http://www.pingdom.com/)"
85.17.156.11 - - [03/Oct/2017:18:58:14 +0000] "GET /get HTTP/1.1" 200 523 "-" "Pingdom.com_bot_version_1.4_(http://www.pingdom.com/)"
```

Comment (mjallday) [reply templates/nginx.conf.j2]: 👍

**Comment by yyunikov** [comment]
@mjallday yes, it is also dependent on elb proxy protocol policy update. these changes are dependent and needs to be done at the same time. tested on development environment by manually changing the config and it worked well for me.

**Comment by max-lobur** [comment]
@yyunikov what version of nginx worked for you?

**Comment by yyunikov** [comment]
@max-lobur tested with `1.12.1`, worked without any issues

**Comment by yyunikov** [comment]
@max-lobur this should be applied after nginx update.

**Comment by yyunikov** [comment]
Was implemented in another PR - https://github.com/verygood-ops/nginx-ssl-router/pull/26

**Pull request closed without merging**